frontend/src/forms/PaymentInvoiceForm.jsx

```diff
@@ -44,10 +44,10 @@ export default function PaymentInvoiceForm({ maxAmount = null, isUpdateForm = fa
         {/* <InputNumber
           className="moneyInput"
           min={0}
-          max={maxAmount}
           controls={false}
-          addonAfter={money.currencyPosition === 'after' ? money.currencySymbol : undefined}
-          addonBefore={money.currencyPosition === 'before' ? money.currencySymbol : undefined}
+          max={maxAmount}
+          addonBefore={money.currencyPosition === 'before' ? money.currencySymbol : null}
+          style={{ textAlign: 'left' }}
```

Comment:
keep this
`          addonAfter={money.currencyPosition === 'after' ? money.currencySymbol : undefined}`

Reply:
do you want me uncomment it ?

Reply:
yes